dolphinscheduler-server/src/test/java/org/apache/dolphinscheduler/server/master/dispatch/host/RefreshResourceTaskTest.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.dolphinscheduler.server.master.dispatch.host;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * RefreshResourceTask test
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class RefreshResourceTaskTest {
+
+  @Mock
+  private LowerWeightHostManager.RefreshResourceTask refreshResourceTask;
+
+  @Test
+  public void testGetHostWeightWithResult() {
```

Comment:
Does this UT test anything :)

Reply:
hi @ruanwenjun 
   updated!

Reply:
Please resolve the code style.

Reply:
And can the latest code go inside the method? It seems you still use @Mock here

Reply:
@ruanwenjun
sorry, fix it, can you help with it ?

Reply:
Please add RefreshResourceTaskTest in pom.xml like other UT file.

Reply:
@ruanwenjun  done